src/fights.ts

```diff
@@ -1762,6 +1837,15 @@ const freeRunFightSources = [
       ],
     }
   ),
+  // Try to accelerate the shadow nc, if you're able to do a quest
+  new FreeRunFight(
+    () =>
+      have($item`closed-circuit pay phone`) &&
+      get("rufusQuestType") !== "items" &&
+      !have($effect`Shadow Affinity`) &&
+      get("encountersUntilSRChoice") > 0,
+    (runSource: ActionSource) => garboAdventure(bestShadowRift(), runSource.macro)
+  ),
```

Comment:
We tend to get shadow affinity when we grab shadow forest items earlier in dailyItems. Should I just remove this code?
```suggestion
```